src/lib/core/BUILD.gn

```diff
@@ -95,6 +95,8 @@ static_library("core") {
     "CHIPKeyIds.h",
     "CHIPTLV.h",
     "CHIPTLVDebug.cpp",
+    "CHIPTLVText.cpp",
```

Comment:
We don't want this in anything that looks like it might be a shipping product, given its assumptions about the trustworthiness of the TLV it's printing.

Reply:
With the fixes to not allocate on stack, is that more amenable?

Reply:
Please move to a separate library. This should not  be in 'core'. 

Then embedded can prove that  they never link it in.